node-stack/miner/miner.docker-compose.yml

```diff
@@ -65,6 +67,19 @@ services:
       - net.ipv4.conf.all.src_valid_mark=1
     restart: unless-stopped
 
+    autoheal:
+      container_name: tpn_validator_autoheal
```

Comment:
Should autoheal be in miner compose and is it using the right container name? @actuallymentor